styles/main/style.css

```diff
@@ -350,6 +352,54 @@ html {
   float: right;
 }
 
+/* Contacts section */
+.contacts {
+  background-color: var(--contacts-dark-gray);
+  margin-top: 140px;
+  text-align: center;
+  padding: 100px 0px;
+  position: relative;
+  z-index: 0;
+}
+.contacts__box{
+  position: relative;
+}
+.contacts__box:before{
+  content: "Hello";
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: var(--contacts-light-gray);
+  font-size: 30vw;
+  letter-spacing: 0.13em;
+  text-align: center;
+  z-index: -1;
+}
```

Comment:
"Hello" extends outside of the parent element, which results in an extra gap in the bottom.

![image](https://user-images.githubusercontent.com/68795596/208678196-a3aa71db-3a61-46fc-ac02-c33cfbdc5281.png)

Reply:
Also, with some screen sizes there is a horizontal overflow

![image](https://user-images.githubusercontent.com/68795596/208678445-1ff27516-9cd3-46ed-85f5-35a04f25c96a.png)